pom.xml

```diff
@@ -376,8 +376,8 @@
         <version>${maven-plugin-tools.version}</version>
         <configuration>
           <goalPrefix>hpi</goalPrefix>
-          <skipErrorNoDescriptorsFound>true</skipErrorNoDescriptorsFound>
-          <!-- see MNG-5346 -->
+          <extractors>java-annotations</extractors>
+          <mojoDependencies></mojoDependencies>
```

Comment:
as all our mojos are not in dependencies tell maven not too look in any of our dependencies (some of this are plugins which is interesting!)

by default this list is `null` which means check everything, and empty list is check no dependencies

Reply:
https://maven.apache.org/plugin-tools/maven-plugin-plugin/descriptor-mojo.html#mojoDependencies

Reply:
> as all our mojos are not in dependencies

Actually one of them is: `jetty-maven-plugin`, needed for `RunMojo` (and why your tests were failing).